Essentials/src/com/earth2me/essentials/commands/Commandfireball.java

```diff
@@ -77,6 +75,8 @@ protected void run(final Server server, final User user, final String commandLab
         Projectile projectile = user.getWorld().spawn(user.getBase().getEyeLocation().add(direction.getX(), direction.getY(), direction.getZ()), types.get(type));
         projectile.setShooter(user.getBase());
         projectile.setVelocity(direction);
+        projectile.setCustomName("Fireball Arrow");
+        projectile.setCustomNameVisible(false);
```

Comment:
I suggest to you to use metadata to check the item is checked correctly that not similar to other items that have "Fireball Arrow" name.